CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 2.19.0-rc.0 / 2020-06-08
+
+* [FEATURE] TSDB: Memory-map full chunks of Head (in-memory) block from disk. This reduces memory footprint and makes restarts faster. #6679
+* [ENHANCEMENT] TODO. #7188
```

Comment:
?

Reply:
I am trying to figure out the exact text to put in there, still a draft :)

Reply:
Hmm, this looks like a cleanup change with no impact in which case we wouldn't list it.

Reply:
That saves time